app/src/main/res/values/strings.xml

```diff
@@ -57,6 +57,7 @@
     <string name="msg_server_profile_added">Server added.</string>
     <string name="msg_server_profile_deleted">Server deleted.</string>
     <string name="msg_pip_not_supported">Picture-in-Picture is not supported on this device</string>
+    <string name="msg_change_ptr_not_supported">Pointer styling not supported on this device</string>
```

Comment:
```suggestion
    <string name="msg_ptr_hiding_not_supported">Hiding cursor is not supported on this device</string>
```

This message should be aligned with the target preference. 'Pointer styling' & 'Cursor hiding' seems unrelated to me.